cpu/ppc64_cpu_test.py

```diff
@@ -62,6 +62,9 @@ def setUp(self):
         self.key = 0
         self.value = ""
         self.max_smt_value = 4
+        if cpu.get_cpu_arch().lower() == 'power9':
+            if 'Hash' in open('/proc/cpuinfo', 'r').read():
```

Comment:
please use `genio` avocado library  for file operation .

code snippet  can be 

```
from avocado.utils import genio
------------------------------------------
-------------------------------------------

if cpu.get_cpu_arch().lower() == 'power9':
     if 'Hash' in genio.read_file('/proc/cpuinfo').rstrip(' \t\r\n\0'):
                             self.max_smt_value = 8

```